build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
 }
 
 plugins {
-    id "com.diffplug.gradle.spotless" version "3.24.2"
+    id "com.diffplug.gradle.spotless" version "3.24.3"
```

Comment:
@ahoffer, I'm starting a thread to respond to https://github.com/connexta/ion-search/pull/35#issuecomment-534650736. What do you mean by "This PR includes a CVE". Where did you see that finding while OWASP succeeded?